server/assets/boards/berlin.json

```diff
@@ -0,0 +1,273 @@
+{
+    "fields":[
+        {
+            "name" : "Start",
+            "type" : "start"
+        },
+        {
+            "name" : "Mahrzahn",
+            "type" : "street",
+            "group" : 0,
+            "price" : 60,
+            "house_price" : 50,
+            "rent" : [2,10,30,90,160,250]
+        },
+        {
+            "name" : "Community Chest",
+            "type" : "society_card"
+        },
+        {
+            "name" : "Mahlsdorf",
+            "type" : "street",
+            "group" : 0,
+            "price" : 60,
+            "house_price" : 50,
+            "rent" : [4,20,60,180,320,450]
+        },
+        {
+            "name" : "Income Tax",
+            "type" : "tax",
+            "rent" : [200]
+        },
+        {
+            "name" : "Westhafen",
+            "type" : "station",
+            "group" : 8,
+            "price" : 200,
+            "rent" : [50]
```

Comment:
rent should be 50, 100, 150, 200

Reply:
Imo it should be the base rent. And the client and server know the rest to calculate the rent for the station.